tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,2 +1,9 @@
-frost-object-details testbed
-{{frost-object-details}}
```

Comment:
Can you throw a title on here so that it's clearer what's going on?  Maybe "User list"?